src/Hl7.FhirPath.Tests/Tests/CastTests.cs

```diff
@@ -50,7 +50,7 @@ public void CastFromNull()
             checkCast<object>(null, null);
             checkCast<IEnumerable<ITypedElement>>(null, ElementNode.EmptyList);
             checkCast<ITypedElement>(null, null);
-            Assert.IsFalse(Typecasts.CanCastTo(null, typeof(bool)));
+            //Assert.IsFalse(Typecasts.CanCastTo(null, typeof(bool)));
```

Comment:
Is this still failing? Otherwise we can turn it on again

Reply:
no longer fails, re-enabled.